course/api/views.py

```diff
@@ -577,3 +580,39 @@ def get_queryset(self) -> QuerySet[News]:
                     Q(audience=AUDIENCE.INTERNAL_USERS)
                 )
         return queryset
+
+class CourseStatisticsViewSet(BaseStatisticsViewSet):
+    """
+    Returns submission statistics for a course, over a given time window.
+
+    Returns the following attributes:
+
+    - `submission_count`: total number of submissions.
+    - `submitters`: number of users submitting.
+
+    Operations
+    ----------
+
+    `GET /courses/<course_id>/statistics/`:
+        returns the statistics for the given course.
+
+    - URL parameters:
+        - `endtime`: date and time in ISO 8601 format indicating the end point
+          of time window we are interested in. Default: now.
+        - `starttime`: date and time in ISO 8601 format indicating the start point
+          of time window we are interested in. Default: one day before endtime
+    """
+    def list(
+            self,
+            request: Request,
+            version: Optional[Union[int, str]] = None,
+            course_id: Optional[Union[int, str]] = None,
+            ) -> Response:
```

Comment:
Adding the parameters `*args` and `**kwargs` is recommended. Our DRF API code has followed that pattern. The code won't crash immediately then if the framework has new parameters in some version.